doc/Analytics.xml

```diff
@@ -3397,127 +3397,71 @@ Added Recognition rule engines</revremark>
     </section>
     <section>
       <title>Counting rule</title>
-      <para>Counting rule counts the number of motion object passing through the set of line segments (barrier) and optional direction attribute. The configuration parameters also include the time interval to report the events and time interval to reset its counter. </para>
+      <para>Counting rule counts the number of objects passing through a set of line segments
+        (barriers) with an optional direction. The configuration parameters also include the time
+        interval to report the events and time interval to reset its counter.<variablelist>
+          <varlistentry>
+            <term>Parameters</term>
+            <listitem>
+              <para role="param">Segments [tt:Polyline]</para>
+              <para role="text">One or more polylines used for detection. An object must have passed
+                through all of them to register as a count. ??? or ??? Passing through any of the
+                lines registers as a count. ???</para>
+              <para>ReportTimeInterval [xs:duration]</para>
+              <para>Optional time interval to report count information, starting from midnight. For
+                example, if the interval is set to PT2H, reports will be published at 2:00 am, 4:00
+                am, etc. Default is same as ResetTimeInterval.</para>
+              <para>ResetTimeInterval [xs:duration]</para>
```

Comment:
How about a mechanism to manually reset the counters?

Reply:
Manual reset would require a function call. I don't see a use case for it, and it complicates this feature a lot. We would have to make a new service for this. Regarding overflow, I prefer to stop.

Reply:
Based on VEWG call  on 11/12, below two options were discussed,

1.) one opinion is to update this as ResetTime, where client can specify actual time in xs:time format, since in options we can specify minoccurrs and maxoccurs, client can know how many reset times are possible. eg: can set the reset time as morning 9 am and 6 pm (to calculate day time visitors and night visitors)
2.) Another opinion is to move this out to a separate api.